app/models/planet.rb

```diff
@@ -5,7 +5,7 @@ class Planet < ApplicationRecord
   validates :capacity, presence: true, numericality: { greater_than: 0 }
   belongs_to :user
   # only 1 environment per planet for now
-  belongs_to :environment
```

Comment:
IMPORTANT TO KEEP IN MIND: This line has been commented out to avoid validations issues, but if we add our environment model later we will need to revisit this.